ConsoleApp04classwork/ConsoleApp04classwork/Program.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+
+namespace ConsoleApp
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine(Max(-1, -2, -3, -4, -5, -6, -7, -2, -10));
+            Console.WriteLine(Min(-1, -2, -3, -4, -5, -6, -7, -2, -10));
+            Console.WriteLine(Max(-1, -2, -3));
+            Console.WriteLine(Max(-1, -2, -3, -4));
+            Console.WriteLine(Min(-1, -2, -3));
+            Console.WriteLine(Min(-1, -2, -3, -4));
+            Console.WriteLine(Repeat("Hello", 3));
+        }
+        static int Max(params int[] args )
+        {
+            int max = args[0];
+            for (int i = 0; i < args.Length; i++)
```

Comment:
it's not critical, but you can start from element with index 1, result will be the same